osu.Game.Tests/Database/BeatmapImporterTests.cs

```diff
@@ -852,7 +852,11 @@ private static void ensureLoaded(Realm realm, int timeout = 60000)
         {
             IQueryable<RealmBeatmapSet>? resultSets = null;
 
-            waitForOrAssert(() => (resultSets = realm.All<RealmBeatmapSet>().Where(s => !s.DeletePending && s.OnlineID == 241526)).Any(),
+            waitForOrAssert(() =>
+                {
+                    realm.Refresh();
```

Comment:
Technically this one should not be necessary because it's the main context, but I suppose it makes sense for safety.

Reply:
This test isn't running a full game loop, so actually..

Reply:
Oh, I see...